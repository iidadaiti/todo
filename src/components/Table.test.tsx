import { render } from '@testing-library/react'
import React from 'react'
import {
  Table,
  TableBody,
  TableCell,
  TableHead,
  TableHeadCell,
  TableRow
} from './Table'

test('Table snapshot', () => {
  const result = render(
    <Table>
      <TableHead>
        <TableRow>
          <TableHeadCell></TableHeadCell>
        </TableRow>
      </TableHead>

      <TableBody>
        <TableRow>
          <TableCell></TableCell>
        </TableRow>
      </TableBody>
    </Table>
  )
  expect(result.asFragment()).toMatchSnapshot()
})
