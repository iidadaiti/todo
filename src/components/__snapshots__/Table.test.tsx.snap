// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table snapshot 1`] = `
<DocumentFragment>
  <table
    class="spectrum-Table spectrum-Table--sizeM"
  >
    <thead
      class="spectrum-Table-head"
    >
      <tr
        class="spectrum-Table-row"
      >
        <th
          class="spectrum-Table-headCell"
        />
      </tr>
    </thead>
    <tbody
      class="spectrum-Table-body"
    >
      <tr
        class="spectrum-Table-row"
      >
        <td
          class="spectrum-Table-cell"
        />
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
